### DISCUSSION TITLE
Add a possibility to delete all Notifications

### DISCUSSION THREAD

**Discussion by kichetof** [laravel/nova-issues#5140] [Ideas & Feature Requests]
Hello,

Actually, you can mark all notification as read, but you can't delete all in one action.
Each deletion need a validation to the alert, that could take long time to delete all.

Hopefully you'll add something to delete all :)

**Comment by Tarpsvo** [comment]
@davidhemphill A quick-win feature. A highly appreciated one from those users who heavily rely on notifications. 🙏

**Comment by davidhemphill** [comment]
I just added this and pushed it in 4.27.8 👍

**Reply by Tarpsvo** [reply]
This is amazing news, thanks a ton! 🥇

**Reply by kichetof** [reply]
Many thanks for this feature!! Really appreciate! 👏
